### ISSUE TITLE
 Cannot generate deps.jl

### ISSUE THREAD

**Issue by CiaranOMara** [bicycle1885/CodecZlib.jl#39]
Build error on FreeBSD 12.0. Likely related to https://github.com/bicycle1885/ZlibBuilder/issues/11.

```bash
(v1.0) pkg> add CodecZlib
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed StatsBase ─ v0.27.0
  Updating `/usr/home/username/.julia/environments/v1.0/Project.toml`
 [no changes]
  Updating `/usr/home/username/.julia/environments/v1.0/Manifest.toml`
  [2913bbd2] ↑ StatsBase v0.26.0 ⇒ v0.27.0

(v1.0) pkg> build CodecZlib
  Building CodecZlib → `~/.julia/packages/CodecZlib/DAjXH/deps/build.log`
┌ Error: Error building `CodecZlib`:
│ ERROR: LoadError: LibraryProduct(nothing, ["libz"], :libz, "Prefix(/home/username/.julia/packages/CodecZlib/DAjXH/deps/usr)") is not satisfied, cannot generate deps.jl!                                           
│ Stacktrace:
│  [1] error(::String) at ./error.jl:33
│  [2] #write_deps_file#152(::Bool, ::Function, ::String, ::Array{LibraryProduct,1}) at /home/username/.julia/packages/BinaryProvider/4F5Hq/src/Products.jl:414                                                      
│  [3] (::getfield(BinaryProvider, Symbol("#kw##write_deps_file")))(::NamedTuple{(:verbose,),Tuple{Bool}}, ::typeof(write_deps_file), ::String, ::Array{LibraryProduct,1}) at ./none:0                             
│  [4] top-level scope at /home/username/.julia/packages/CodecZlib/DAjXH/deps/build.jl:93
│  [5] include at ./boot.jl:317 [inlined]
│  [6] include_relative(::Module, ::String) at ./loading.jl:1044
│  [7] include(::Module, ::String) at ./sysimg.jl:29
│  [8] include(::String) at ./client.jl:392
│  [9] top-level scope at none:0
│ in expression starting at /home/username/.julia/packages/CodecZlib/DAjXH/deps/build.jl:78
└ @ Pkg.Operations /home/vagrant/worker/package_freebsd64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1097                                                                                             

(v1.0) pkg> 
```

```bash
julia> versioninfo()
Julia Version 1.0.3
Commit 099e826241 (2018-12-18 01:34 UTC)
Platform Info:
  OS: FreeBSD (x86_64-unknown-freebsd11.1)
  CPU: AMD Opteron 63xx class CPU
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, bdver1)
```

```bash
uname -a
FreeBSD hostname.novalocal 12.0-RELEASE FreeBSD 12.0-RELEASE r341666 GENERIC  amd64
```

**Comment by bicycle1885** [comment]
I think this is also resolved by #40. If you still see the problem, please let me know.

**Comment by CiaranOMara** [comment]
I think I originally encountered this error when trying to use [BGZFStreams](https://github.com/BioJulia/BGZFStreams.jl).
